wdl-lint/src/rules/line_width.rs

```diff
@@ -89,8 +83,7 @@ impl Rule for LineWidthRule {
     }
 
     fn description(&self) -> &'static str {
-        "Ensures that lines do not exceed a certain width. That width is currently set to 90 \
-         characters."
+        "Ensures that lines do not exceed a certain width."
```

Comment:
The default should be described in the explanation (which it is), so it can be removed here.